### ISSUE TITLE
POST /login response

### ISSUE THREAD

**Issue by herikudaru** [Arcue/Webshop_userDatabase#14]
Dokumentationen säger att json responsens key är "x-auth-token" egentliga key är "x_auth_token". På frontenden har jag för tillfället den och fungera tillsammans med "x_auth_token". Tänker ni ändra respons eller dokumentation?

**Comment by JolleJ** [comment]
> Dokumentationen säger att json responsens key är "x-auth-token" egentliga key är "x_auth_token". På frontenden har jag för tillfället den och fungera tillsammans med "x_auth_token". Tänker ni ändra respons eller dokumentation?

Lättast från min sida är att ända apin till x_auth_token då jag inte får postgres att acceptera namn med bindestreck såsom x-auth-token

**Comment by JolleJ** [comment]
> Dokumentationen säger att json responsens key är "x-auth-token" egentliga key är "x_auth_token". På frontenden har jag för tillfället den och fungera tillsammans med "x_auth_token". Tänker ni ändra respons eller dokumentation?

Yes fixar det nu på en gångs så att token kommer som x-auth-token

**Comment by JolleJ** [comment]
> Dokumentationen säger att json responsens key är "x-auth-token" egentliga key är "x_auth_token". På frontenden har jag för tillfället den och fungera tillsammans med "x_auth_token". Tänker ni ändra respons eller dokumentation?

Lättast för mig är att ändra dokumentationen då C# och postgres inte tycker om variabel namnet x-auth-token
